packages/table-react/src/TableRow.tsx

```diff
@@ -14,6 +14,7 @@ export interface TableAnchorRowData extends BaseTableRowData {
     href: string;
     hrefLabel: string;
     onRowClick?: (href: string) => void;
+    customRowClick?: () => void;
```

Comment:
I'll second @Saegrov here. In general it's not a great pattern to have one prop invalidate the intent of another, as this will increase the complexity of the component and make it harder to use and understand.

This looks fine
```
...
 onRowClick={()=>console.log('the row were clicked')}
 customRowClick={()=>console.log('custom row')}
...
```

but onRowClick will never do anything..
```